espnet2/asr/espnet_model.py

```diff
@@ -10,6 +10,7 @@
 from espnet2.asr.decoder.abs_decoder import AbsDecoder
```

Comment:
I'm concerning that `espnet2/asr/espnet_model.py` becomes very big.

As @ftshijt suggested, if we make a new task for slu (e.g., `espnet2/slu/espnet_model.py`), we can customize it without caring about the ASR module. Can you consider it?